### ISSUE TITLE
Add Arrow output serialization.

### ISSUE THREAD

**Issue by niviksha** [uwdata/arquero#31]
@jheer and others, this is already looking like a great library for us at OmniSci, where we just added the ability to return SQL query results over the wire as Arrow buffers 👍 👍 

Wondering if it is possible to serialize the results of an Arquero query to Arrow - it would make for a composable pipeline e.g. for use cases where you retrieve a dataset from OmniSci, wrangle it client-side with Arquero, and then either visualize it with Vega-based charts or things like deck.gl for 3d charts, for example.

We're happy to help in any way we can!

**Comment by bmschmidt** [comment]
👍  would use for a slightly different case: it would let you more easily do arquero computations in a web worker, which is where I do most of my Arrow computation when not on Observable.

**Comment by jheer** [comment]
I've started a notebook for prototyping and testing Arrow serialization here:
https://observablehq.com/@jheer/arquero-to-arrow-serialization

Feedback and suggestions are welcome!

Also, @bmschmidt, in case you haven't seen it yet, you might be interested in https://github.com/uwdata/arquero-worker, which implements Arquero worker thread support. Right now the inter-thread communication is done via JSON string serialization, but the plan has always been to eventually augment that with a more efficient transfer options.

**Comment by jheer** [comment]
I've released a new package - [arquero-arrow](https://github.com/uwdata/arquero-arrow) - that supports Arrow serialization of Arquero tables as well as arrays of JavaScript objects. This new package provides a simple interface to the apache-arrow JavaScript library (on which it depends), while also using higher-performance encoders for standard integer, float, date, boolean, and string dictionary types.

I also updated the [Arquero and Apache Arrow notebook](https://observablehq.com/@uwdata/arquero-and-apache-arrow) to demonstrate Arrow serialization!

cc @niviksha, @bmschmidt, @ericemc3
